src/vanilla/Templates/ServiceClientTemplate.cshtml

```diff
@@ -26,7 +26,10 @@ const ServiceClient = msRest.ServiceClient;
 }
 
 @EmptyLine
-const models = require('./models');
+@if (Model.HasModelTypes())
```

Comment:
My POV on this is just stick with ModelTypes.Any() especially since there's a .Any() right below this

Reply:
Sure. I can do that.